src/swell/deployment/create_experiment.py

```diff
@@ -236,7 +262,11 @@ def copy_eva_files(logger, swell_suite_path, exp_suite_path):
 # --------------------------------------------------------------------------------------------------
 
 
-def copy_platform_files(logger, exp_suite_path, platform=None):
+def copy_platform_files(
+    logger: 'Logger',
```

Comment:
Why a string here rather than the `Logger` object?

```suggestion
    logger: Logger,
```

Reply:
No reason other than that I was in the habit of it, I've changed this and all other forward references to Logger.